<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>12</number>
         </property>
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <item row="0" column="2">
          <widget class="QLabel" name="titleLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Helvetica Neue</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>No file loaded</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="authorLabel">
           <property name="font">
            <font>
             <family>Sans</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Select a file through File -&gt; Open</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item row="0" column="4" rowspan="2">
          <widget class="QToolButton" name="toolButton_4">
           <property name="text">
            <string>-&gt;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" rowspan="2">
          <widget class="QFrame" name="frame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0" rowspan="2">
          <widget class="QToolButton" name="toolButton_5">
           <property name="text">
            <string>&lt;-</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3" rowspan="2">
          <widget class="QLabel" name="timeLabel">
           <property name="text">
            <string>0:00 / 0:00</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QToolButton" name="playButton">
           <property name="text">
            <string>&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="pauseButton">
           <property name="text">
            <string>||</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="stopButton">
           <property name="text">
            <string>O</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="seekBar">
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="invertedControls">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="seekLeft">
           <property name="text">
            <string>&lt;&lt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="seekRight">
           <property name="text">
            <string>&gt;&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="volumeSlider">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Playlist</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QToolButton" name="addToPlaylist">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="playlistView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Info</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>537</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
